### DISCUSSION TITLE
Refused connection to 127.0.0.1:8000 in ch3-postgresql in DJANGO for PROFESSIONALS

### DISCUSSION THREAD

**Discussion by eddhyne** [community/community#66357] [Programming Help]
### Body

I am having a problem in ch3-postgressql
When I run python manage.py runserver  I get:
[docker error.docx](https://github.com/community/community/files/12549792/docker.error.docx)
it responds with http://127.0.0.1:8000/ and if I hold down the Ctrl key and click on it takes me to the rocket ship in my Microsoft Edge browser. Great 127.0.0.1: 8000 works.

If I run docker build . then docker-compose up -d it responds with {+] Running 1/1   then   
 Container ch3-postgresql-web-1 Started
When I enter 127.0.0.1:8000 in the same browser, I get 127.0.0.1 I get: 
[docker error.docx](https://github.com/community/community/files/12549835/docker.error.docx)
refused to connect.
[docker error.docx](https://github.com/community/community/files/12549846/docker.error.docx)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by eddhyne** [comment]
I tried to add the following attachment
![image](https://github.com/community/community/assets/92999884/9f17b66d-139b-4501-ba43-1e7714401cca)

**Comment by seyLu** [comment]
#### Hi @eddhyne 

Assign IP address to 0.0.0.0 instead of 127.0.0.1.

You can read more at [Docker server IP address difference between 127.0.0.1 and 0.0.0.0](https://stackoverflow.com/questions/59179831/docker-app-server-ip-address-127-0-0-1-difference-of-0-0-0-0-ip).

**Comment by eddhyne** [comment]
Thanks.

Ed Dhyne
Direct: 954-608-6603
***@***.******@***.***>
www.linkedin.com
